elixir/apps/domain/lib/domain/config/definitions.ex

```diff
@@ -290,16 +292,6 @@ defmodule Domain.Config.Definitions do
   """
   defconfig(:database_password, :string, default: nil, sensitive: true)
 
-  @doc """
-  Replication user that will be used to connect to replication slots.
-  """
-  defconfig(:database_replication_user, :string, default: nil, sensitive: true)
-
-  @doc """
-  Replication password for the replication user.
-  """
-  defconfig(:database_replication_password, :string, default: nil, sensitive: true)
-
```

Comment:
Did we end up abandoning the idea of using a separate user for the replication connection?

Reply:
Yeah we did, reasons:

- both users would essentially have access to the same data from the same application/context
- to create the replication slot and publication in the manager GenServer, we need the same privs as the firezone user (same for migrations etc)
- We create the replication slot and publications at application start as opposed to having them all the time so that we can (1) test them, and (2) ensure they slot isn't active until a `domain` application starts

Reply:
Makes sense